src/components/Api.js

```diff
@@ -0,0 +1,105 @@
+export default class Api {
+  constructor({ baseUrl, headers }) {
+    this._baseUrl = baseUrl;
+    this._headers = headers;
+  }
+
+  getInitialCards() {
+    return fetch(`${this._baseUrl}/cards`, {
+      method: "GET",
+      headers: this._headers,
+    }).then((res) => {
```

Comment:
Needs correction: The code for processing the response is duplicated in all of the api methods. You should create a separate method `_processResponse` and use it instead. 

Other api methods will look like the following:

```js
return fetch(...).then(this._processResponse)
```